src/utils/supports.js

```diff
@@ -25,11 +26,13 @@ const platformFeatures = {
   '@pleroma/local-visibility-post': containPleroma,
   '@akkoma/local-visibility-post': containAkkoma,
 };
+
 const supportsCache = {};
 
 function supports(feature) {
   try {
-    const { version, domain } = getCurrentInstance();
+    let { version, domain, software_name } = getCurrentInstance();
```

Comment:
If nodeinfo object is stored separately, possibly need another `getCurrentNodeInfo` function. So this could look like this:

```
const { software: { name: softwareName } } = getCurrentNodeInfo();
```